src/app/components/BibPage/BibDetails.jsx

```diff
@@ -77,20 +84,19 @@ class BibDetails extends React.Component {
         {
           bibValues.map((value, i) => {
             const url = `filters[${fieldValue}]=${value['@id']}`;
-            return (
-              <li key={i}>
-                {
-                  fieldLinkable ?
-                    <Link
-                      onClick={e => this.newSearch(e, url)}
-                      to={`${appConfig.baseUrl}/search?${url}`}
-                    >
-                      {value.prefLabel}
-                    </Link>
-                    : <span>{value.prefLabel}</span>
-                }
-              </li>
-            );
+            let itemValue = fieldLinkable ?
```

Comment:
Does this mean `itemValue` will have three different kinds of value. A link to `fieldSelfLinkable `, a link to search and a label without link.